tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -23,9 +23,10 @@
     ArgMaxClassificationPrediction
 )
 
-from ..config import NumberField, StringField, ConfigError
+from ..config import NumberField, StringField, ConfigError, BoolField
 from .metric import PerImageEvaluationMetric
 from .average_meter import AverageMeter
+from sklearn.metrics import accuracy_score
```

Comment:
please make sklearn optional or do not use it at all for such simple metric.
it is problematic dependency for some platforms

Reply:
fixed